cinema/views.py

```diff
@@ -48,6 +78,36 @@ class MovieSessionViewSet(viewsets.ModelViewSet):
     queryset = MovieSession.objects.all()
     serializer_class = MovieSessionSerializer
 
+    def get_queryset(self):
+        queryset = self.queryset
+
+        date = self.request.query_params.get("date")
+        movie = self.request.query_params.get("movie")
+
+        if date:
+            date = datetime.strptime(date, "%Y-%m-%d")
+            queryset = queryset.filter(
+                show_time__year=date.year,
+                show_time__month=date.month,
+                show_time__day=date.day
```

Comment:
You already have data in the format needed.
Read [checklist](https://github.com/mate-academy/py-tickets-orders/blob/master/checklist.md)

Reply:
My bad, thanks)